www/js/album.js

```diff
@@ -1,14 +1,51 @@
+let albumDialogOpen;
+
 $$(document).on('page:init', '.page[data-name="album"]', function (e) {
   var albumId = e.detail.route.params.albumId;
+  albumDialogOpen = false;
   $.getJSON(API + '/albums/' + albumId)
     .done(function(resp) {
       initAlbums(resp.album);
     })
     .fail(function(resp) {
-      console.log(resp.statusText);
+      $$('.album-content .preloader').remove();
+      if (resp.statusText === 'Unauthorized') {
+        albumDialog.open();
+      } else {
+        $$('.album-content').append('<div class="block">Kunde inte läsa in album</div>');
+      }
     });
 });
 
+const albumDialog = app.dialog.create({
+  title: 'Fel',
+  text: 'Du är inte auktoriserad för att se albumet. Detta beror på att ditt medlemskap inte är bekräftat.',
+  closeByBackdropClick: false,
+  buttons: [
+    {
+      text: 'Ok',
+      onClick: function() {
+        alternativesView.router.back();
+      }
+    }
+  ],
+  on: {
+    close: function() {
+      albumDialogOpen = false;
+    },
+    open: function() {
+      albumDialogOpen = true;
+    }
+  }
+});
+
+function albumBackButton() {
+  if (albumDialogOpen) {
+    albumDialog.close();
```

Comment:
You can access the currently open dialog through `app.dialog` and then close it like normal. This would remove the need for the dialog to be globally defined. 
```suggestion
    app.dialog.close();
```